pyproject.toml

```diff
@@ -22,6 +22,7 @@ dependencies = [
     "tqdm>=4.66.3",
     "pytest>=7.1.3",
     "pytest-pycharm>=0.7.0",
+    "astropy>=6.0.0",
```

Comment:
Added astropy because it includes a reader for the `ipac` data format. This makes it easy to convert that format to Pandas.